templates/about/index.html

```diff
@@ -81,7 +81,7 @@ <h3>认证的 Ubuntu 分支版本</h3>
         <li class="p-list__item is-ticked"><a href="https://ubuntu-mate.org/">Ubuntu MATE</a> — 具有 MATE 桌面环境的 Ubuntu</li>
       </ul>
       <h3>本地化</h3>
-      <p>出于空间方面的考虑，标准 Ubuntu 映像仅随附了少数几种语言。用户可以通过系统设置或使用网络安装器来添加其他语言的支持。标准映像直接提供对一下相关语言的完整支持。</p>
+      <p>出于空间方面的考虑，标准 Ubuntu 映像仅随附了少数几种语言。用户可以通过系统设置或使用网络安装器来添加其他语言的支持。标准映像直接提供对以下相关语言的完整支持。</p>
```

Comment:
```suggestion
      <p>出于空间方面的考虑，标准 Ubuntu 映像仅随附了少数几种语言。用户可以通过系统设置或使用网络安装器来添加其他语言的支持。标准映像直接提供对以下相关语言的完整支持：</p>
```
@zprood How about this?

Reply:
I think we need to create a new list for this. Thank you for reporting this.